### PR TITLE
Give example for what a webfinger is

### PR DESCRIPTION
**Pull request by orblivion**
I was stuck at this part for a while. I had to read through mediagoblin docs a bit for it to click to try this. Since webfinger isn't a common concept, I thought an example was in order.

### CODE REVIEW

**Comment by SapienTech** [comment]
Hey there, one of the devs may be able to give a better answer, but a webfinger is a unique id for your account on a particular pump enabled website. For instance, if I have an account on a site with the web address example.net, then my webfinger will be account_name@example.net. Hope that helps!

**Comment by orblivion** [comment]
Right, I figured that out. This is a pull request to put it in the README :)

**Comment by SapienTech** [comment]
o right, lol, thought this was an issue :)

File: README.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ To add a new pump.io account to p you need to authorize it:
 $ p authorize <webfinger>
 ```
 
+_example:_
+
+```
+$ p authorize my_user@mediagoblin.example.com
```
Comment (moggers87) [inline README.md]: "user@example.com" would be better IMO - p is a pump.io client not mediagoblin specific.

**Comment by orblivion** [comment]
@moggers87 How about now?

**Comment by orblivion** [comment]
I used "user@server.tld" because it was used in a later example in the same README file. I missed it before. Judging by the fact that I missed it, I think having it higher up in the file as well would be better.

**Comment by moggers87** [comment]
Oops. this PR has been sitting here doing nothing for too long.

**Pull request merged**